setup.py

```diff
@@ -124,15 +125,14 @@ def build_extension(self, ext):  # noqa C901
             ["cmake", "--build", "."] + build_args, cwd=build_temp, check=True
         )
 
-
 setup(
     name="pytorch-tokenizers",
     version="0.1.0",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/meta-pytorch/tokenizers",
     packages=find_packages(),
-    ext_modules=[CMakeExtension("pytorch_tokenizers_cpp")],
+    ext_modules=[CMakeExtension("pytorch_tokenizers.pytorch_tokenizers_cpp")],
```

Comment:
What is this for? Is it another windows nuance?

Reply:
From what I understand, this is the way it should be, as it handles the namespacing correctly. The extension name should be the fully qualified name of the extension module. We were doing something slightly weird in that we were updating the CMake build to tell it to dump the library into a different directory. But with this change, it just works on all platforms.